Projection Storer Configurations improvement (#1244)
mia-platform/documentation: docs/fast_data/configuration/projection_storer.md

#### Kafka Configuration

> If you select the Kafka Consumer, the service will create a Kafka Client to consume messages from the ingestion topics you setup in the _Projections_ section.

Reviewer: ```suggestion
If you select the Kafka Consumer, the service will create a Kafka Client to consume messages from the ingestion topics you setup in the _Systems of Record_ section.
```
The section is now named `Systems of Record`

> If you select the Kafka Consumer, the service will create a Kafka Client to consume messages from the ingestion topics you setup in the _Systems of Record_ section.

Reviewer 2: ```suggestion
If you select the Kafka Consumer, the service will create a Kafka Client to consume messages from the ingestion topics you setup.
```

#### Kafka Configuration

> If you select the Kafka Consumer, the service will create a Kafka Client to produce messages into the topics related to the PR Updates you setup.

Reviewer: ```suggestion
If you select the Kafka Producer, the service will create a Kafka Client to produce messages into the topics related to the PR Updates you setup.
```

> If you select the Kafka Consumer, the service will create a Kafka Client to produce messages into the topics related to the PR Updates you setup in the _Systems of Record_ section.

Reviewer 2: ```suggestion
If you select the Kafka Consumer, the service will create a Kafka Client to produce messages into the topics related to the PR Updates you setup.
```
